src/plugin/__tests__/elementSpec.spec.ts

```diff
@@ -58,7 +58,7 @@ describe("mount", () => {
           // field1 is derived from the fieldDescriptions, and is a string b/c it's a richText field
           fields.field1.toString();
           // field2 is a boolean b/c it's a checkbox field
-          fields.field2.value.valueOf();
+          fields.field2;
```

Comment:
This no longer tests the boolean-ness of the field – we want something that'll exist on a Boolean, but no other type. IIRC valueOf was such a method.